bingo.qmd

```diff
@@ -0,0 +1,48 @@
+---
+title: "Tidyverse developer day bingo"
+format: pdf
+---
+
+Welcome to tidyverse developer day! To help get to know each other, we'll start the day off with some bingo. It will go like this:
+
+-   Mingle and find people who match the descriptions on your card.
+
+-   When you find a match, write the person's name in the corresponding square. (Only one square per person!)
+
+-   The goal is to fill as many squares as possible. If you're the first to fill in the whole card, come and find Simon to be crowned the winner!
```

Comment:
I love this overall idea! But can we tweak it slightly so that we can have multiple "winners"? Then we can start it in parallel as people arrive.